streams/src/test/java/org/apache/kafka/streams/state/internals/InMemoryLRUCacheStoreTest.java

```diff
@@ -44,7 +45,7 @@
             Deserializer<K> keyDeser = (Deserializer<K>) context.keyDeserializer();
             Serializer<V> valSer = (Serializer<V>) context.valueSerializer();
             Deserializer<V> valDeser = (Deserializer<V>) context.valueDeserializer();
-            supplier = Stores.create("my-store").withKeys(keySer, keyDeser).withValues(valSer, valDeser).inMemory().maxEntries(10).build();
+            supplier = Stores.create("my-store").withKeys(SerDes.serialization(keySer, keyDeser)).withValues(SerDes.serialization(valSer, valDeser)).inMemory().maxEntries(10).build();
```

Comment:
Would it make sense to unify Sers and Desers here, too?  Why must we pass them separately?

```java
.withKeys(SerDes.serialization(keySer, keyDeser)).withValues(SerDes.serialization(valSer, valDeser))
```